### PR TITLE
Document Firefox and Safari behavior wrt createConicGradient

### PR DESCRIPTION
**Pull request by wbamberg**
See https://github.com/mdn/content/pull/22363 - when gradients are created using [`CanvasRenderingContext2D.createConicGradient()`](https://developer.mozilla.org/en-US/docs/Web/API/CanvasRenderingContext2D/createConicGradient) Firefox and Safari start the gradient at 12 o'clock, but the standard now expects them to start at 3 o'clock, like the other Canvas APIs.

2 questions:
- I'm not sure the wording here is the best. Maybe "12 o'clock" would be better?
- per https://github.com/mdn/content/pull/22363#issuecomment-1324388656, it looks like Safari has updated to the standard, but AFAICT this isn't in a released version yet. But I'm hoping someone can tell me if I have to do something else here.

### CODE REVIEW

**Comment by queengooborg** [comment]
Would a subfeature be better in this case?

**Comment by wbamberg** [comment]
What are the criteria for when to make something a subfeature? To me this PR just documents a way that these browsers have different behavior from the spec. It's not about support/non-support for a discrete aspect of a feature.

File: api/CanvasRenderingContext2D.json [hunk 1]
```diff
@@ -409,7 +409,8 @@
             "chrome_android": "mirror",
             "edge": "mirror",
             "firefox": {
-              "version_added": "90"
+              "version_added": "90",
+              "notes": "The gradient starts from a line going vertically up from the center, like the equivalent CSS function."
```
Comment (Elchi3) [inline api/CanvasRenderingContext2D.json:413]: I think we should make it clear that there is a different implementation going on. Then, when fixed in a later version, we can have a new range that used version_added without partial implementation. 
```suggestion
              "notes": "Implements an older version of the specification. The gradient starts from a line going vertically up from the center, like the equivalent CSS function.",
              "partial_implementation": true
```

File: api/CanvasRenderingContext2D.json [hunk 2]
```diff
@@ -419,7 +420,8 @@
             "opera": "mirror",
             "opera_android": "mirror",
             "safari": {
-              "version_added": "15"
+              "version_added": "15",
+              "notes": "The gradient starts from a line going vertically up from the center, like the equivalent CSS function."
```
Comment (Elchi3) [inline api/CanvasRenderingContext2D.json:424]:
```suggestion
              "notes": "Implements an older version of the specification. The gradient starts from a line going vertically up from the center, like the equivalent CSS function.",
              "partial_implementation": true
```

**Comment by Elchi3** [comment]
> Would a subfeature be better in this case?

No, I think a note and partial implementation is fine here.

**Comment by wbamberg** [comment]
Actually it looks like Firefox just started being compliant https://bugzilla.mozilla.org/show_bug.cgi?id=1711171

**Comment by Elchi3** [comment]
Then we probably want a range here. Seems like the fix was in Firefox 112 https://bugzilla.mozilla.org/show_bug.cgi?id=1818911

Safari is compliant too https://bugs.webkit.org/show_bug.cgi?id=244254 

https://wpt.fyi/results/html/canvas/element/fill-and-stroke-styles?label=experimental&label=master&aligned&view=subtest&q=canvas%20conic

It was in TP 154 https://webkit.org/blog/13207/release-notes-for-safari-technology-preview-154/
I guess TP 154 is included in the release that followed: 16.1 https://webkit.org/blog/13399/webkit-features-in-safari-16-1/

File: api/CanvasRenderingContext2D.json [hunk 3]
```diff
@@ -408,19 +408,33 @@
...
             },
             "oculus": "mirror",
             "opera": "mirror",
             "opera_android": "mirror",
-            "safari": {
-              "version_added": "15"
-            },
+            "safari": [
+              {
+                "version_added": "16.4"
```
Comment (Elchi3) [inline api/CanvasRenderingContext2D.json:430]:
```suggestion
                "version_added": "16.1"
```

**Review by Elchi3 — APPROVED** [review]
Dragging this over the finish line :)

**Pull request merged**